etc/php/tools/prepare_tests

```diff
@@ -1,5 +1,7 @@
 #!/bin/bash
 
+set -xe
```

Comment:
It is not necessary, it is a duplicate of https://github.com/duhon/magento-docker/blob/storefront/etc/php/tools/magento#L18